R/zzz.R

```diff
@@ -10,6 +10,7 @@ on_load({
 
 .onLoad <- function(lib, pkg) {
   run_on_load()
+  ark_register_methods()
```

Comment:
I suspect there's a reason why we can't `on_load({ ark_register_methods() })` . Is it worth adding a comment here?